libvast/src/meta_index.cpp

```diff
@@ -49,11 +49,12 @@ operator[](const uuid& partition) const {
 void meta_index::add(const uuid& partition,
                      const const_table_slice_handle& slice) {
   auto& rng = partitions_[partition].range;
+  VAST_DEBUG(this, "meta_index got table slice:", slice);
```

Comment:
This doesn't read like a proper sentence.

Reply:
done